.gitignore

```
@@ -169,8 +169,15 @@ src/reportImport/agent/reportImport
 src/delagent/agent_tests/Unit/test_delagent
 src/delagent/agent_tests/Unit/testconf/
 /utils/fo_unicode_clean
+<<<<<<< HEAD
 install/db/db.cron
 install/fossdash/fossdash-publish.py
+=======
+install/fossdash/fossdash-publish.py
+install/db/db.cron
+>>>>>>> origin/dev/reuse-software-agent/main
```

```suggestion
install/fossdash/fossdash-publish.py
install/db/db.cron
```
I think you forgot to solve this conflict 😅

---

oops, thanks @avinal